test/workload-checks/typical/cases/apm_app_server/experiment.yaml

```diff
@@ -0,0 +1,72 @@
+description: >
+  Simulates a relatively busy application server on which DogStatsD metrics, APM
+  traces and TCP streamed logs are present on which the client user has mostly
+  transitioned to the use of APM over DogStatsD and TCP listener logs, although
+  not entirely replacing these sources of metrics and logs. Traces represent the
+  majority of load. We make claims about throughput, UDS packet loss and memory,
+  CPU resource consumption.
+
+teams:
+  - agent-core
```

Comment:
There was previously an agent team called "agent-core" -- this has now split into "agent-shared-components" and my team "agent-metrics-logs". I assume this is just saying "this is a core agent check", but maybe a different name would be less confusing? Or maybe I'm overthinking, feel free to ship as-is.

Reply:
Ah yeah, and these aren't hooked up properly yet. I will drop this.

Reply:
Addressed in [b09e52b](https://github.com/DataDog/datadog-agent/pull/18814/commits/b09e52bbc04d7901437d78dd2e3934031d3f383d)